server/bin/deploy

```diff
@@ -286,6 +288,23 @@ else
     HOSTEDTEST=""
 fi
 
+
+if [ "$QPID" == "qpid" ]; then
+    info_msg "Enable Qpid"
+    QPID="qpid=yes"
+
+    #Setup testing qpid queue allmsg, this will be needed by Qpid spec tests
+    QPID_KEYS_DIR=$PROJECT_DIR/../gutterball/bin/qpid/keys
+    # Delete the allmsg first, just to make the script indempotent. The error messages produced during
+    # deletion are silenced
+    qpid-config -b amqps://localhost:5671 --ssl-certificate $QPID_KEYS_DIR/qpid_ca.crt --ssl-key $QPID_KEYS_DIR/qpid_ca.key --force del queue allmsg 2 > /dev/null || true
+    qpid-config -b amqps://localhost:5671 --ssl-certificate $QPID_KEYS_DIR/qpid_ca.crt --ssl-key $QPID_KEYS_DIR/qpid_ca.key add queue allmsg --durable
+    qpid-config -b amqps://localhost:5671 --ssl-certificate $QPID_KEYS_DIR/qpid_ca.crt --ssl-key $QPID_KEYS_DIR/qpid_ca.key bind event allmsg "#"
+else
+    QPID=""
+fi
+
```

Comment:
@mstead good point. What I can do is to try delete the queue every time we use deploy script. In case it fails just silently ignore that failure. What do you think? Another approach might be to create the queue non-durable. Such queue would be deleted upon qpid restartup

I am fine with both approaches